lua/telescope/_extensions/zoxide/list.lua

```diff
@@ -104,6 +104,7 @@ return function(opts)
 
   pickers.new(opts, {
     prompt_title = z_config.get_config().prompt_title,
+    previewer = require("telescope.previewers.buffer_previewer").cat.new(opts),
```

Comment:
Could you move the require on top of the file and use that here?

like:

**Top of file:**
```diff
local actions = require("telescope.actions")
local action_state = require("telescope.actions.state")
local finders = require("telescope.finders")
local pickers = require("telescope.pickers")
local sorters = require("telescope.sorters")
+ local previewers = require("telescope.previewers")
local utils = require("telescope.utils")
```

**Usage:**
```diff
pickers
  .new(opts, {
    prompt_title = z_config.get_config().prompt_title,
    + previewer = previewers.vim_buffer_cat.new(opts),

    finder = finders.new_table({
```